<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border-radius:5px;
  background-color: white; 
  color: #00cec9; 
  border: 2px solid #00cec9;
}

.QPushButton:hover {
  background-color: #00cec9;
  color: white;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegis">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border-radius:5px;
  background-color: white; 
  color: #00cec9; 
  border: 2px solid #00cec9;
}

.QPushButton:hover {
  background-color: #00cec9;
  color: white;
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border: 2px solid #00b894;
background-color: #55efc4;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPass">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border: 2px solid #00b894;
background-color: #55efc4;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#00b894;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
